docs/monoDrive_home/IMU.md

```diff
@@ -1,7 +1,8 @@
 # IMU
 
-## IMU
-Provides IMU stream
+Provides inertial measurements EGO vehicle relative to the sensor's location.
```

Comment:
Provides inertial measurements **for the** EGO vehicle relative to the sensor's location.

Reply:
Fixed.